methods/eps_fhir/api_request_body_generators.py

```diff
@@ -0,0 +1,516 @@
+import json
+import uuid
+
+
+def create_fhir_resource(resource_type, main_keys, **kwargs):
```

Comment:
I think this should be split into separate functions for bundle and parameters.

Also, there's a lot of **kwargs unpacking in this file, that doesn't gain us anything. Can these be changed to set params please?

Also also, there's https://pypi.org/project/fhir.resources/ that might help with your fhir bits and pieces.

Reply:
The initial approach was indeed having two separate functions which didn't make the code DRY so we consolidated them. We could look into splitting them again.
I do understand your comment about the kwargs, however, in the current state they allow us to be flexible, they are still readable and they dont really hurt us. 
Again I think depends on what @seansteberisal would like to do with this.

Reply:
I think Bundle and Parameters are distinct enough to be handled in separate functions, yes.

The kwargs don't really offer us much flexibility over normal params, when those main_keys are defined in the function. The code would read better if we weren't iterating through main_keys to get at the values in the kwargs.

Again, I think it's really worth looking at what that fhir.resources package might offer here. If we're going to be adding more items to these generators, it could really help.